instructure/canvas-ios
# Migrate to string catalog

### What changed?
- Migrated all .strings and .stringsdict files to the new xcstrings files.
- Updated the translation export script to handle the singe file export of all translations of the workspace instead of separate files for each project.
- Updated CI to work with the new script.
- Updated xcodegen config to have the proper build settings for string generation.
- Manually fixed a few translations.
- Added verbatim formatting option to some preview related code where translation is not needed.
- Removed unnecessary translations of infoplist files for internal projects.
- Removed translation of the CoreTester project.
- Disabled Indonesian language support as there were only a few strings translated and web doesn't support it.
- Import script update is to be done after we receive the translations with the new format.
- Translation import workflow trigger on CI was disabled for now.

refs: MBL-17197
affects: Student, Teacher, Parent
release note: none

test plan:
- Check out the branch, explore the new xcstrings files.
- Use the file filter during PR review and filter to .swift and .js files.
- Change language in your canvas profile on web, check if translations still work after logging out/in in the apps.

## Checklist
